src/VolumeClaimCommands.cpp

```diff
@@ -88,9 +88,15 @@ crow::response listVolumeClaims(PersistentStore& store, const crow::request& req
 		} 
 
 		// Add volume status from K8s (Bound, Pending...)
+		if((volumeStatus.IsObject() && volumeStatus.HasMember("status"))
+		   && volumeStatus["status"].IsObject() && volumeStatus["status"].HasMember("phase")){
 		volumeData.AddMember("status", volumeStatus["status"]["phase"], alloc);
```

Comment:
This line should be indented by one more tab.

Reply:
Oh I forgot to indent, okay that makes sense. So something more like this?
```cpp
                // Add volume status from K8s (Bound, Pending...)
		if((volumeStatus.IsObject() && volumeStatus.HasMember("status"))
	  	   && volumeStatus["status"].IsObject() && volumeStatus["status"].HasMember("phase")){
			volumeData.AddMember("status", volumeStatus["status"]["phase"], alloc);
		}
		else{
			volumeData.AddMember("status", "unknown", alloc);
		}
		volumeResult.AddMember("metadata", volumeData, alloc);
		resultItems.PushBack(volumeResult, alloc);
```

Reply:
Will this prevent status "unknown" clusters from breaking `slate cluster list`? Or is there something else I should add?

Reply:
That indentation now looks correct. 

This doesn't change anything with respect to `slate cluster list`, since this specific problem did not affect that command.

Reply:
Ah I meant volume list, my bad.

Reply:
In my testing this seems to be sufficient, although as mentioned in the issue thread it can still be extremely slow due to having to wait for the timeout on each inaccessible volume. It may still be best to merge this change first to fix the outright bug, and address the quality of implementation in a later patch.

Reply:
Okay sounds like a good idea, I'll open a ticket in order to skip inaccessible volumes. Thanks for the help